client/apps/employee/src/pages/DailyReportPage.tsx

```
@@ -241,6 +268,32 @@ export const DailyReportPage: React.FC = () => {
+            <header className="daily-report-page__header">
+                <h1 className="daily-report-page__title">דיווח שעות</h1>
+                <div className="daily-report-page__month-nav">
+                    <button
+                        className="daily-report-page__nav-btn"
+                        onClick={handlePrevMonth}
+                        aria-label="חודש קודם"
+                    >
+                        <svg width="7" height="12" viewBox="0 0 7 12" fill="none">
+                            <path d="M1 1L6 6L1 11" stroke="currentColor" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round"/>
+                        </svg>
+                    </button>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)